web_environment_ribbon/README.rst

```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
```

Comment:
Should be https://www.gnu.org/licenses/agpl as is defined in https://github.com/OCA/maintainer-tools/edit/master/template/module/README.rst ?